Engine/Core/Event.cpp

```diff
@@ -342,6 +345,12 @@ void ONScripter::waitEvent(int count, bool nopPreferred) {
 		}
 
 		while (true) {
+#if defined(DISCORD)
+			auto it = ons.ons_cfg_options.find("discord");
```

Comment:
Finding an option is fairly expensive. Please introduce a bool in ONScripter.hpp instead.